frontend/src/scenes/pipeline/pipelineNodeNewLogic.tsx

```diff
@@ -42,11 +49,20 @@ export const pipelineNodeNewLogic = kea<pipelineNodeNewLogicType>([
                     path: urls.pipeline(stage ? NODE_STAGE_TO_PIPELINE_TAB[stage] : undefined),
                 },
                 {
-                    // TODO: use the plugin name
                     key: pluginId || batchDestination || 'Unknown',
-                    name: pluginId ? pluginId.toString() : batchDestination ?? 'Options',
+                    name: pluginId ? 'New' : batchDestination ? `New ${batchDestination} destination` : 'Options',
                 },
             ],
         ],
+        batchExportServiceNames: [
+            (s) => [s.user],
+            (user): BatchExportService['type'][] => {
+                const services: BatchExportService['type'][] = ['BigQuery', 'Postgres', 'Redshift', 'Snowflake', 'S3']
+                if (user?.is_impersonated || user?.is_staff) {
```

Comment:
I feel like this needs a comment as I don't understand why staff or impersonated users get http?

Reply:
will do in a follow-up PR - I'll make sure it matches the backend comment too.